### ISSUE TITLE
CommentsIndentation Check false-positive on a method chain

### ISSUE THREAD

**Issue by Vladlis** [checkstyle/checkstyle#3217]
Input:
```java
class Input {
    void foo() {
        getClass()
            .getName()
            .isEmpty();
            // comment
    }
}
```
Output:
```
vlad@vlad-N61PC-M2S ~/cli $ java -jar checkstyle-6.18-all.jar -c my_check.xml Test.java
Starting audit...
[ERROR] /home/vlad/cli/Test.java:6: Comment has incorrect indentation level 12, expected is 8, indentation should be the same level as line 3. [CommentsIndentation]
Audit done.
Checkstyle ends with 1 errors.
```
Config:
```xml
<module name = "Checker">
    <module name="TreeWalker">
         <module name="CommentsIndentation"/>
    </module>
</module>
```
Expected - no violations.

**Comment by romani** [comment]
@Vladlis , can you make a link to real code.

For now I am not agree on expected result.  As comment is on new line and after ";" it could be a comment for end of block(the same indentation as "getClass()") and ".isEmpty()".

Current behaviour is ok.

**Comment by Vladlis** [comment]
@romani, I cannot provide a real code for now, but if you say
>  it could be a comment for end of block(the same indentation as "getClass()") and ".isEmpty()".

then what indentation do you expect, if it actually a comment for `.isEmpty()` ? I suppose that it should be at the same level as `.isEmpty()` in this case, so the input I provide should be correct.

**Comment by romani** [comment]
Sorry. Only one expected indentation - the same as "getClass()".

**Comment by romani** [comment]
@Vladlis , should we close the issue ?

**Comment by Vladlis** [comment]
> should we close the issue ?

I'm not sure. Where would you put a comment for `isEmpty()`?

**Comment by romani** [comment]
above ".isEmpty()", comment is always above a commenting code.

**Comment by Vladlis** [comment]
ok
